nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/java/org/apache/nifi/json/TestInferJsonSchemaAccessStrategy.java

```diff
@@ -41,32 +44,32 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class TestInferJsonSchemaAccessStrategy {
-    private final String dateFormat = RecordFieldType.DATE.getDefaultFormat();
-    private final String timeFormat = RecordFieldType.TIME.getDefaultFormat();
-    private final String timestampFormat = "yyyy-MM-DD'T'HH:mm:ss.SSS'Z'";
+class TestInferJsonSchemaAccessStrategy {
```

Comment:
What is the reason for removing the `public` modifier on the class and methods? The vast majority of test classes have the public modifier, so recommend reverting the change.

Reply:
Since the JUnit5 upgrade public modifiers are not necessary. I'd leave it like this it it's okay.